Sources/Core/GTMSessionFetcher.m

```diff
@@ -2600,7 +2611,7 @@ - (void)invokeOnCallbackQueue:(dispatch_queue_t)callbackQueue
                         block:(void (^)(void))block {
   if (callbackQueue) {
     dispatch_group_async(_callbackGroup, callbackQueue, ^{
-      if (!afterStopped) {
+      if (!afterStopped && !self->_stopFetchingTriggersCallbacks) {
```

Comment:
I worry this will let other blocks slip through if `_stopFetchingTriggersCallbacks` is true, and that may not be what we want.

For Swift `async` my understanding is the only callback we want firing after `stopFetching` is called is the completion handler because that's the one that needs to trigger the continuation. Are we sure we want to allow other callback blocks to be sent? Otherwise, I think the `afterStopped` should be what we adjust if `.stopFetchingTriggersCallbacks` is set.

Reply:
I couldn't find a way to get the afterStopped parameter right. I did update the added test to limit to when triggersCompletion is on and there is a userStop occurrence.

Reply:
Does this still need work or did this get figured out?

Reply:
I've updated to limit the fall through to only when there has been a stop, so yes.